_problems/fee-bumping.md

```diff
@@ -23,30 +25,33 @@ Let's look at the implications of the current methods have on layer-2 protocols.
 
 It is much more desirable to allow the party to spontaneously increase the fee at will.
 This can be done with [CPFP].
-For example, in the [Lightning] protocol commitment transactions are designed with [anchor outputs] so they can be easily fee bumped through CPFP.
+For example, in the [Lightning] protocol commitment transactions have [anchor outputs] so they can be easily fee bumped through CPFP.
+Although this works it is rather complex to design and implement.
+Furthermore, users have to keep unattached fee-bumping UTXOs around in their wallet in case their need to do a fee-bump via CPFP.
 
 ### Resource Usage of Replacement
 
-Allowing a mempool transaction to be replaced by higher fee paying transactionrmay open up Bitcoin network nodes to *denial of service* (DoS) attacks.
+Allowing a mempool transaction to be replaced by higher fee paying transaction may open up Bitcoin network nodes to *denial of service* (DoS) attacks.
 Without replacement, a malicious party's ability to get your node to process a transaction is limited by the number of UTXOs they own.
-With replacement an attacker can repeatedly spam your node with transactions that replace previous ones even if they only own a single UTXO.
+With replacement an attacker can repeatedly spam your node with transactions that replace previous ones even if they only own a single UTXO forcing you to spend time verifying them.
 
 To prevent effective denial of service attacks, replacement transactions have to follow some rather strict rules as defined in [BIP125].
-In particular a replacement transaction must pay a higher absolute fee and a higher feerate than the transactions they are evicting and it must not evict more than 100 transactions.
+In particular a replacement transaction must pay a higher absolute fee (rule 3) and a higher feerate (rule 4) than the transaction it is evicting.
+In addition, it must not evict more than 100 transactions (rule 5).
 
 For layer-2 protocols these rules had the unintended consequence of creating "pinning" attacks (see [Related Research](#related-research)).
 For example, a malicious party could add 100 descendants to a transaction that conflicts with the one the honest party needs confirm.
-Even if the honest party pays a large fee it still won't evict the malicious party's transaction since that would evict more than 100 transactions.
-The other [BIP125] rules may be abused as well.
+Even if the honest party pays a large fee it still won't evict the malicious party's transaction since that would evict more than 100 transactions which breaks rule 5.
+Rule 3 may be abused as well by creating a very low feerate (so miners won't include it) but high absolute fee (so it won't be evicted) transaction that conflicts with the honest party's transaction.
```

Comment:
"Note, that the block-to-propagate pre-signed transaction absolute fee is known by the attacker, and can even be maliciously lowered if the attacker is the funder of the targeted Lightning channel :/ "

Reply:
I'm patiently waiting for your mailing post about replace by feerate (i.e. drop rule 3) :) I think we can omit discussion of all the different situations that rule 3 causes problems because it can be removed without any concern that I can see.